front/components/assistant/conversation/SidebarMenu.tsx

```diff
@@ -84,10 +84,13 @@ export function AssistantSidebarMenu({ owner }: { owner: WorkspaceType }) {
               href={`/w/${owner.sId}/assistant/new`}
               onClick={() => {
                 setSidebarOpen(false);
-                if (
-                  router.pathname === "/w/[wId]/assistant/new" &&
-                  triggerInputAnimation
-                ) {
+                const { cId } = router.query;
+                const isNewConversation =
+                  router.pathname === "/w/[wId]/assistant/[cId]" &&
+                  typeof cId === "string" &&
+                  cId === "new";
+
+                if (isNewConversation && triggerInputAnimation) {
```

Comment:
UX-wise it's a bit weird to me that if you're on the homepage and click on "New Conversation" we reload the page which leads to a new greeting message

Reply:
Yes will see with @Duncid